README.md

```diff
@@ -277,6 +277,147 @@ MotionAnimator is a purely Core Animation-based animator. If you are looking for
 - [POP](https://github.com/facebook/pop)
 - [CADisplayLink](https://developer.apple.com/documentation/quartzcore/cadisplaylink)
 
+# Core Animation: a deep dive
+
+> Recommended reading:
+>
+> - [Building Animation Driven Interfaces](http://asciiwwdc.com/2010/sessions/123)
+> - [Core Animation in Practice, Part 1](http://asciiwwdc.com/2010/sessions/424)
+> - [Core Animation in Practice, Part 2](http://asciiwwdc.com/2010/sessions/425)
+> - [Building Interruptible and Responsive Interactions](http://asciiwwdc.com/2014/sessions/236)
+> - [Advanced Graphics and Animations for iOS Apps](http://asciiwwdc.com/2014/sessions/419)
+> - [Advances in UIKit Animations and Transitions](http://asciiwwdc.com/2016/sessions/216)
+> - [Animating Layer Content](https://developer.apple.com/library/content/documentation/Cocoa/Conceptual/CoreAnimation_guide/CreatingBasicAnimations/CreatingBasicAnimations.html)
+> - [Advanced Animation Tricks](https://developer.apple.com/library/content/documentation/Cocoa/Conceptual/CoreAnimation_guide/AdvancedAnimationTricks/AdvancedAnimationTricks.html)
+> - [Additive animations: animateWithDuration in iOS 8](http://iosoteric.com/additive-animations-animatewithduration-in-ios-8/)
+
+There are two primary ways to animate with Core Animation on iOS:
+
+1. **implicitly**, with the UIView `animateWithDuration:` APIs, or by setting properties on unhosted CALayer instances, and
```

Comment:
Unhosted CALayers? If I'm not mistaken the examples show setting properties on _hosted_ CALayer instances implicitly animates.

Reply:
Clarified this.